sdk/loadtesting/load-testing-rest/samples-dev/addAppComponents.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to add App Components to an existing test.
+ *
+ * @summary Demonstrates how to add app components to an existing test.
+ */
+
+import AzureLoadTesting, { isUnexpected } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+
+async function main(): Promise<void> {
+  /**
+   * The dataplane endpoint for the Azure Load Testing resource.
+   * Refer to https://learn.microsoft.com/rest/api/loadtesting/data-plane-uri to understand how to obtain the data-plane endpoint.
+   */
+  const endpoint = process.env["LOADTESTSERVICE_ENDPOINT"] || "";
+
+  /** Microsoft Entra ID authentication */
+  /**
+   * In this sample you can populate the three AZURE_CLIENT_ID, AZURE_CLIENT_SECRET & AZURE_TENANT_ID variables for Microsoft Entra ID auth
+   */
+  const credential = new DefaultAzureCredential();
+  const testId = process.env["LOADTESTSERVICE_TESTID"] || ""; // TestId of a test already created.
+
+  const SUBSCRIPTION_ID = process.env["SUBSCRIPTION_ID"] || "";
+
+  // Build a client through AAD
+  const client = AzureLoadTesting(endpoint, credential);
+
+  const appComponentCreationResult = await client
+    .path("/tests/{testId}/app-components", testId)
```

Comment:
I understand testId is in path, but still TM needs testId in body too, ref : 
![Uploading image.png…]()

Reply:
![image](https://github.com/user-attachments/assets/bb145f2a-8423-4bfb-a9c7-c9726f6bea02)

Reply:
It is not needed in the body, its optional